### PR TITLE
Tackles

### PR DESCRIPTION
**Pull request by stackmm**
1) refactored season.rb and league.rb (and associated test files) to pull data from all three csv files
2) created `most_tackles` method in season (and associated tests)
3) created the following helper methods needed for `most_tackles` (and probably other methods): 
- `create_season`: returns an array with just the lines of data (ie, the games) associated with a specific year/season)
- `get_team_name` : returns the team name when given the team_id (as a string)

### CODE REVIEW

File: lib/season.rb [hunk 1]
```diff
@@ -1,8 +1,46 @@
...
-  def initialize(data)
-    @season_data = data
+  def initialize(games_data, games_teams_data, teams_data)
+    @games_data = games_data
+    @games_teams_data = games_teams_data
+    @teams_data = teams_data
   end
-  
+
+  # creates an array with just the games(rows) from a specific season
```
Comment (boomclear) [inline lib/season.rb]: can we remove these comments before we merge?

**Review by boomclear — CHANGES_REQUESTED** [review]
Remove the comments in season.rb and I think we are good to go to merge this.

**Review by Jesse193 — APPROVED** [review]
good for merge, test pass

**Review by GlowMunch — APPROVED** [review]
comments removed, test passes

**Review by boomclear — APPROVED** [review]

**Comment by boomclear** [comment]
thanks for the changes

**Pull request merged**